### PR TITLE
France overseas

### PR DESCRIPTION
**Pull request by kent1D**
We also should add french overseas territories.

There are a lot for UK, why not France ;)

Otherwise, avoid some territories and islands without label at all (French Guyana, Reunion, New Caledonia...)

### CODE REVIEW

File: data/country.csv [hunk 1]
```diff
@@ -1,218 +1,226 @@
...
-Deutschland,Allemagne,,Germany,ألمانيا,1,1-PME/FE,51.0834196,10.4234469,DE,
-España,Espagne,,Spain,إسبانيا,1,1-PME/FE,40.0028028,-4.003104,ES,
-United States of America,États-Unis,,United States of America,الولايات المتّحدة الأمريكيّة,1,1-PME/FE,39.7837304,-100.4458825,US,
-Italia,Italie,,Italy,إيطاليا,1,1-PME/FE,42.6384261,12.674297,IT,
-Madagascar,Madagascar,,Madagascar,مدغشقر,1,1-PME/FE,-18.9249604,46.4416422,MG,
-Maroc ⴰⵎⵔⵔⵓⴽ المغرب,Maroc,,Morocco,المغرب,1,1-PME/FE,31.1728192,-7.3366043,MA,
-United Kingdom,Royaume-Uni,,United Kingdom,المملكة المتحدة,1,1-PME/FE,54.7023545,-3.2765753,GB,
-Sénégal,Sénégal,,Senegal,السنغال,1,1-PME/FE,14.4750607,-14.4529612,SN,
+name,name:fr,short_name:fr,name:en,name:ar,prio,type,latitude,longitude,iso,sov,wikidata,admin_level
+La Réunion,La Réunion,,Réunion,ريونيون,99,99-Other,-21.13093,55.52658,RE,FR,Q17070,4
```
Comment (yohanboniface) [inline data/country.csv]: I feel like this is data pâté ;)
Country are by definition admin_level=2, mixin them with other territories would need at least a rename.
But the first of all why not using OSM data directly for those labels?

File: labels.mss [hunk 2]
```diff
@@ -1,11 +1,12 @@
 #country[prio<=3][zoom=3],
 #country[prio<=4][zoom=4],
 #country[zoom>=5][zoom<=10] {
-  text-name: '[name]';
+  text-transform: uppercase;
```
Comment (yohanboniface) [inline labels.mss]: My previous remark about uppercase not being compatible with other alphabet is still valid ;)

File: data/country.csv (hunk 1 above)
Comment (kent1D) [reply data/country.csv]: The fact is that UK's overseas are admin_level=2 and not french ones... Even if french ones have their own ISO code also.

What I wanted is also to show out easlily from which sovereign country it depends (UK / FR / US / NO / DK).

File: labels.mss (hunk 2 above)
Comment (kent1D) [reply labels.mss]: Ok let's abandon it for the moment until I receive a complaint

**Comment by kent1D** [comment]
This PR is now outdated.

Have a look at https://github.com/tilery/mae-boundaries/pull/7 in the other repository

**Pull request closed without merging**